react-basic-practice/src/components/ListProducts/index.jsx

```diff
@@ -39,14 +47,34 @@ const ListProducts = (props) => {
           )
           : (
             <Grid columns="3" rowGap="large" columnGap="large">
-              {products.map((item) => (
+              {products && products.map((item) => (
                 <Link to={`/products/${item.id}`} key={item.id}>
                   <ProductCard product={item} />
                 </Link>
               ))}
             </Grid>
           )
       }
+      {
```

Comment:
Why don't you just do it once here?
{
<Button
              type="button"
              onClick={handleLoadMore}
              className="btn-primary btn-large"
              text="load more"
              status={isDisabled}
            />
}